lips/lip-10.md

```diff
@@ -0,0 +1,191 @@
+---
+lip: 10
+title: Libra ID Spec
+authors: Andrey Chursin(@andll), Kevin Hurley (@kphfb)
+status: Draft
+type: Informational
+created: 11/03/2020
+---
+# Summary
+
+This LIP describes Libra ID - the human-readable identifier for user accounts.   
+
+In addition to describing the format of the Libra ID, this LIP also describes the off-chain protocol for the peer-2-peer payment with Libra ID.
+
+This document does not describe an off-chain protocol for using Libra ID for merchant use case - this protocol will be part of merchant flow LIP.  
+
+# Motivation
+Libra ID provides a convenient way to identify user's account within a VASP. There are two main use cases for Libra ID:
+* For peer-2-peer payments, Libra ID allows a user to provide a human-readable ‘address’ to the sender. Basically, instead of sending a lip-5 address(that needs to be refreshed for each payment, ideally), the receiver can just share their Libra ID once. In this case Libra ID plays the role of an email address for payments. Benefits for this use case:
+  * Libra ID is a user-friendly identifier, unlike lip-5 address
+  * Libra ID resolution process can allow to exchange information about both sender and receiver for payment
+  * Libra ID resolution allows to fetch details about each participant, like profile picture
```

Comment:
I think this one is basically the same as above unless I'm misunderstanding the above